packages/composite-checkout/src/components/checkout-payment-methods.tsx

```diff
@@ -1,3 +1,4 @@
+import { useShoppingCart, hasFreeCouponTransfersOnly } from '@automattic/shopping-cart';
```

Comment:
I mentioned this in Slack but for completeness: I don't think it's a good idea to add a dependency on the shopping-cart package in this package.

The `@automattic/composite-checkout` package is a set of building blocks used to build a checkout-like experience. We use it in calypso for our checkout flow as well as our payment method management pages. It might in the future be used for other things as it is a public npm package. It isn't aware of wpcom or calypso specific things.